boards/nucleo144-f446/include/periph_conf.h

```diff
@@ -29,6 +29,7 @@ extern "C" {
  * @name    Clock system configuration
  * @{
  */
+#define P                       (4U)
```

Comment:
what is this about?

Reply:
I see. A comment on this line would not hurt... But I think this will be obsolete through #7477 in the near future anyway, right? Still, I would prefer to move this line down to the end of the clock config block.